core/src/main/java/org/stellar/anchor/asset/AssetResponse.java

```diff
@@ -104,6 +104,11 @@ public String toString() {
       String name;
 
       String description;
+
+      public DeliveryMethod(String name, String description) {
```

Comment:
Commenting here because I can't comment on the `class AssetResponse` line.

Should we rename this class to `Asset`, `AssetConfig`, or `AssetInfo` since it is not used an HTTP response?

Reply:
Great suggestion! Done in https://github.com/stellar/java-stellar-anchor-sdk/pull/95/commits/e7058c2b5e6e0bf9458c297165df41d4fea88eea.